### ISSUE TITLE
[BUG] go-table without pagination does not render all the data

### ISSUE THREAD

**Issue by eligolding** [mobi/goponents#215]
**Describe the bug**
When setting pagination on the `GoTableConfig` to false, it removes the pagination from the bottom of the table, but does not render all the records. It only renders the first 10 (which is the default page size)

**To Reproduce**
1. Render a go-table with more than 10 records, and set pagination to `false`.
```js
this.tableConfig = new GoTableConfig({
  tableData: data, // array with more than 10 records
  totalCount: data.length,
  pageable: false
});
```
2. Try to find the 11th record. Spoiler alert: you won't!

**Expected behavior**
It should just render all records. This may not be ideal UX, but given that we allow it, it should function properly.

**Screenshots**
The following is from go-tester page 1. I set pageable to `false`. The mock data has 1000 records, but it is only rendering 10
<img width="1388" alt="GoTester 2019-10-02 10-10-58" src="https://user-images.githubusercontent.com/5261826/66051469-faac5400-e4fc-11e9-906d-c7b233cbc9dc.png">

**Comment by patryk0605** [comment]
@grahamhency I'll take this

**Comment by grahamhency** [comment]
Closed with #352
